### PR TITLE
Amf

### PR DESCRIPTION
**Pull request by MrToino**
Added basic python server for the plotter engine.

### CODE REVIEW

**Review by luisferreira32 — CHANGES_REQUESTED** [review]
do a `git pull origin main --rebase`

File: service/plotter/server.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+class MyServer(BaseHTTPRequestHandler):
+    def do_GET(self):
+        self.send_response(200)
+        self.send_header("Content-type", "text/html")
+        self.end_headers()
+        self.wfile.write(bytes("<html><head><title>https://pythonbasics.org</title></head>", "utf-8"))
+        self.wfile.write(bytes("<body>", "utf-8"))
+        self.wfile.write(bytes("<p>Plotter Engine endpoint</p>", "utf-8"))
+        self.wfile.write(bytes("</body></html>", "utf-8"))
```
Comment (luisferreira32) [inline service/plotter/server.py:15]: Could you not send the whole body encoded in one message?

**Review by luisferreira32 — APPROVED** [review]
Just a final nit :+1:

File: service/plotter/server.py (hunk 1 above)
Comment (MrToino) [reply service/plotter/server.py:15]: Yes we can, however, just for an initial POC don't really require such detail.

**Pull request merged**